quantum/nodes/quantum-register/quantum-register.js

```diff
@@ -112,8 +117,10 @@ module.exports = function(RED) {
       // then send one qubit object per node output
       await shell.execute(script, (err) => {
         if (err) node.error(err);
-        else send(output);
       });
+      // wait for quantum circuit to be initialised
+      await circuitReady();
```

Comment:
I believe the `flow` variables could be deleted from the editor here, once `await circuitReady()` has returned.